diff -uNr apcupsd-3.14.4/src/drivers/pcnet/pcnet.c apcupsd-3.14.4.new/src/drivers/pcnet/pcnet.c
--- apcupsd-3.14.4/src/drivers/pcnet/pcnet.c	2007-10-06 16:52:52.000000000 +0200
+++ apcupsd-3.14.4.new/src/drivers/pcnet/pcnet.c	2008-12-14 10:35:02.240131748 +0100
@@ -208,7 +208,7 @@
       break;
    case CI_RUNTIM:
       Dmsg1(80, "Got CI_RUNTIM: %s\n", value);
-      ups->TimeLeft = atof(value);
+      ups->TimeLeft = atof(value) / 60;
       break;
    case CI_ITEMP:
       Dmsg1(80, "Got CI_ITEMP: %s\n", value);
