--- apcupsd-3.8.0/etc/apcupsd.conf.fred	Fri Oct  6 11:25:43 2000
+++ apcupsd-3.8.0/etc/apcupsd.conf	Mon Dec  4 11:03:43 2000
@@ -106,7 +106,7 @@
 # If you want the last few EVENTS to be available over the network
 # by the network information server, you must define an EVENTSFILE.
 # Only the last 50 or so events are kept.
-EVENTSFILE /etc/apcupsd/apcupsd.events
+EVENTSFILE /var/lib/apcupsd/apcupsd.events
 #
 #
 #
@@ -117,7 +117,7 @@
 STATTIME 0
 #
 # Location of STATUS file (written to only if STATTIME is non-zero)
-STATFILE /etc/apcupsd/apcupsd.status
+STATFILE /var/lib/apcupsd/apcupsd.status
 #
 #
 # LOGSTATS [ on | off ] on enables, off disables
