https://bugzilla.redhat.com/show_bug.cgi?id=1053324

--- apcupsd-3.14.13/src/drivers/snmplite/asn.cpp~	2014-04-06 18:40:11.000000000 +0300
+++ apcupsd-3.14.13/src/drivers/snmplite/asn.cpp	2015-09-16 16:46:03.485622689 +0300
@@ -26,6 +26,8 @@
 #include <stdlib.h>
 #include <stdio.h>
 
+extern int debug_level;
+
 using namespace Asn;
 
 // *****************************************************************************
@@ -71,7 +73,9 @@
       obj = new Sequence(type);
       break;      
    default:
-      printf("UNKNOWN ASN type=0x%02x\n", type);
+      if (debug_level) {
+         printf("UNKNOWN ASN type=0x%02x\n", type);
+      }
       obj = NULL;
       break;      
    }
